admin_interface/locale/fr/LC_MESSAGES/django.po

```diff
@@ -174,10 +178,30 @@ msgstr "code"
 msgid "display"
 msgstr "affichage"
 
+#: admin_interface/models.py
+msgid "Required fields, error messages"
+msgstr "Défaut: #333 (champs requis, messages d'erreur)"
```

Comment:
Should not give more information in a translation than the original string

Reply:
Changed ! Thanks for feedback !